.github/workflows/main-docker-alpine.yml

```diff
@@ -16,7 +16,7 @@ env:
   DOCKERHUB_REGISTRY: docker.io
   IMAGE_NAME: ${{ github.repository }}
   TEST_TAG: triliumnext/notes:test
-  PLATFORMS: linux/amd64,linux/arm64
+  PLATFORMS: linux/arm64,linux/arm/v7
```

Comment:
🤔

Reply:
🤦‍♂️

Reply:
I probably should've renamed those `ALPINE_PLATFORMS` and `DEBIAN_PLATFORMS` respectively for our sanity. Copy and pasting all around clearly wasn't my brightest idea.